Like SciPy and matplotlib, this project abuses build requirements to specify
desired versions for building wheels.

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -125,7 +125,7 @@
     'Cython>=3.0.4',
     'pythran',
     'lazy_loader>=0.3',
-    'numpy>=2.0.0rc1',
+    'numpy',
 ]
 
 [tool.spin]
